app/src/test/java/org/dhis2/usescases/login/LoginViewModelTest.kt

```diff
@@ -62,13 +64,12 @@ class LoginPresenterTest {
     private val analyticsHelper: AnalyticsHelper = mock()
     private val crashReportController: CrashReportController = mock()
     private val network: NetworkUtils = mock()
+    private lateinit var loginViewModel: LoginViewModel
+    private lateinit var loginViewModelWithNullUserManager: LoginViewModel
```

Comment:
I think that you don't need two variables as is going to be assigned once per test, you can reuse loginViewModel